modules/cnf-understanding-workload-hints.adoc

```diff
@@ -62,3 +62,122 @@ perPodPowerManagement: true
     | Allows for power management per pod.
 
 |===
+
+This table describes the list of kernel arguments configured by applying the workload hints. The kernel arguments apply to all profiles with the exceptions called out in notes in the description field.
+
+[cols="33%,67",options="header"]
+|===
+| Argument | Description 
+
+| `skew_tick=1`
+a|Configures the kernel to adjust the timer skew. This parameter adjusts the timing of timer interrupts to synchronize with the CPU frequency and reduce timing errors.
+
+
+| `tsc=reliable`
+a| Specifies that the timestamp counter (TSC) should be used as a reliable time source. The TSC is a high-resolution per-CPU counter used for performance monitoring and timing purposes.
+
+
+| `rcupdate.rcu_normal_after_boot=1`
+a| This setting ensures that read-copy update (RCU) `normal` state is deferred until after the system boot process completes, potentially optimizing boot performance and avoiding interference with other boot-time tasks.
+
+| `nohz=on`
+|This setting enables the `NOHZ` feature in the Linux kernel, which reduces timer interrupts on idle CPUs to improve power efficiency and overall system performance.
+
+
+| `rcu_nocbs=3-15`
+| Specifies that CPUs 3 through 15 (inclusive) are excluded from acting as "no-CBs" CPUs. This means that these CPUs will not be responsible for executing RCU callbacks. Instead, other CPUs in the system will handle RCU-related tasks. By specifying the CPUs that handle RCU callbacks, you can better use CPU resources and improve overall system performance.
```

Comment:
```suggestion
| `rcu_nocbs=<list of isolated cpus>`
| Specifies that the isolated cpus are excluded from acting as "no-CBs" CPUs. This means that these CPUs will not be responsible for executing RCU callbacks. Instead, other CPUs in the system will handle RCU-related tasks. By specifying the CPUs that handle RCU callbacks, you can better use CPU resources and improve overall system performance.
```
The list of reserved CPUs will (obviously) vary.  
Suggest we document it this way, to better explain how a system will be configured